tests/test_backend.py

```diff
@@ -163,6 +215,75 @@ def test_cache_value_error(self):
         with self.assertRaises(ValueError):
             b.cache = 8
 
+    def test_fetch_client_not_provided(self):
+        """Test whether an NotImplementedError exception is thrown"""
+
+        b = Backend('test')
+
+        with self.assertRaises(NotImplementedError):
+            _ = [item for item in b.fetch(category="acme")]
+
+    def test_fetch_items(self):
+        """Test whether an NotImplementedError exception is thrown"""
+
+        b = Backend('test')
+
+        with self.assertRaises(NotImplementedError):
+            b.fetch_items()
+
+    def test_init_client(self):
```

Comment:
This isn't a valid test because it might be failing because `fetch_items` is not implemented. Call to `_init_client` instead.